Ionic 2 rc3 error "Can't bind to 'options'"

Everything worked fine, after update gives this error.
```
Can't bind to 'options' since it isn't a known property of 'div'. ("
                <ion-item>
                    <div ngFileDrop
                         [ERROR ->][options]="profileOptions"
```
template.html
```html
                        <div ngFileDrop
                             [ngClass]="{'file-over': protectionDropZoneOver}"
                             [options]="protectionOptions"
                             (onUpload)="handleUpload($event)"
                             (onFileOver)="fileOverBaseProtection($event)"
                             class="image-drop-zone">
```

`app.module.ts`
```typescript
import { Ng2UploaderModule } from 'ng2-uploader';
...
@NgModule({
  imports: [
    IonicModule.forRoot(MyApp),
    Ng2UploaderModule,
    BrowserModule,
    FormsModule,
    MomentModule,
    AngularFireModule.initializeApp(firebaseConfig, myFirebaseAuthConfig),
  ],
....
```

just unpacked it and ran
`npm install`
`npm run serve`

and got same error
```
vendor.js:7698 Unhandled Promise rejection: Template parse errors:
Can't bind to 'options' since it isn't a known property of 'div'. ("
               *ngIf="!response"
               ngFileDrop
               [ERROR ->][options]="options"
               (onUpload)="handleUpload($event)"
               [ngClass]="{'file"): AppDemoComponent@8:15
Can't bind to 'options' since it isn't a known property of 'input'. ("
                     class="hidden" 
                     ngFileSelect
                     [ERROR ->][options]="options" 
                     (onUpload)="handleUpload($event)">
              Browse
"): AppDemoComponent@18:21 ; Zone: <root> ; Task: Promise.then ; Value: Error: Template parse errors:(…) Error: Template parse errors:
Can't bind to 'options' since it isn't a known property of 'div'. ("
               *ngIf="!response"
               ngFileDrop
               [ERROR ->][options]="options"
               (onUpload)="handleUpload($event)"
               [ngClass]="{'file"): AppDemoComponent@8:15
Can't bind to 'options' since it isn't a known property of 'input'. ("
                     class="hidden" 
                     ngFileSelect
                     [ERROR ->][options]="options" 
                     (onUpload)="handleUpload($event)">
              Browse
"): AppDemoComponent@18:21
    at TemplateParser.parse (http://localhost:4200/vendor.js:34730:19)
    at RuntimeCompiler._compileTemplate (http://localhost:4200/vendor.js:44520:51)
    at http://localhost:4200/vendor.js:44440:62
    at Set.forEach (native)
    at RuntimeCompiler._compileComponents (http://localhost:4200/vendor.js:44440:19)
    at createResult (http://localhost:4200/vendor.js:44336:19)
    at ZoneDelegate.invoke (http://localhost:4200/vendor.js:7542:26)
    at Zone.run (http://localhost:4200/vendor.js:7424:43)
    at http://localhost:4200/vendor.js:7812:57
    at ZoneDelegate.invokeTask (http://localhost:4200/vendor.js:7575:35)consoleError @ vendor.js:7698
vendor.js:7700 Error: Uncaught (in promise): Error: Template parse errors:(…)
```

--- a/src/directives/ng-file-drop.ts
+++ b/src/directives/ng-file-drop.ts
@@ -22,14 +22,14 @@ export class NgFileDropDirective {
 
    _options:any;
 
-  get options(): any {
-    return this._options;
-  }
-
   @Input('options')
   set options(value: any) {
     this._options = value;
     this.uploader.setOptions(this.options);
+  }
+
+  get options(): any {
+    return this._options;
   }
 
   files: any[] = [];

--- a/src/directives/ng-file-select.ts
+++ b/src/directives/ng-file-select.ts
@@ -21,14 +21,14 @@ export class NgFileSelectDirective {
 
   _options: any;
 
-  get options(): any {
-    return this._options;
-  }
-
   @Input('options')
   set options(value: any) {
     this._options = value;
     this.uploader.setOptions(this.options);
+  }
+
+  get options(): any {
+    return this._options;
   }
 
   files: any[] = [];

